index.html

```diff
@@ -64,32 +74,44 @@ <h2 class="header-section-heading-2">Software Developer</h2>
   </section>
     <!-- Middle Section -->
 
-  <div class="skills-section">
-    <section class="ellipse-2"></section>
-    <ul class="language-list-1">
-      <li class="col-header"><h3> Language </h3></li>
-      <li class="list-items list-items-1">Javascript</li>
-      <li class="list-items list-items-1">Ruby</li>
-      <li class="list-items list-items-1">Html</li>
-      <li class="list-items list-items-1">CSS</li>
-    </ul>
-    <ul class="language-list-2">
-      <li class="col-header"><h3>Frameworks</h3></li>
-      <li class="list-items list-items-2">Bootstrap</li>
-      <li class="list-items list-items-2">Ruby on Rails</li>
-      <li class="list-items list-items-2">RSPec</li>
-      <li class="list-items list-items-2">CapyBara</li>
-      <li class="list-items list-items-2">Selenium</li>
-    </ul>
-    <ul class="language-list-3">
-      <li class="col-header"><h3>Skills </h3></li>
-      <li class="list-items list-items-3">Codekit</li>
-      <li class="list-items list-items-3">Guthub</li>
-      <li class="list-items list-items-3">Codepen</li>
-      <li class="list-items list-items-3">Gitlab</li>
-      <li class="list-items list-items-3">Terminal</li>
-    </ul> 
-  </div>
+    <section id="frame_container">
+      <article class="frame">
+        <section class="languages">
+          <h3>Languages</h3>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </section>
+
+        <div class="back"></div>
+        
+        <section class="frameworks">
+          <h3>Frameworks</h3>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </section>
+      </article>
+
+      <article class="skills">
+        <h3>Skills</h3>
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>GitLab</li>
+          <li>Terminal</li>
+        </ul>
+      </article>
+    </section>
```

Comment:
- Please note that the icons on your page are broken, kindly fix them. 👍🏽
![Screenshot from 2022-09-09 22-45-13](https://user-images.githubusercontent.com/44624138/189450852-7aa8c62d-6fc7-4701-b3c7-7d699124c919.png)
![Screenshot from 2022-09-09 22-44-11](https://user-images.githubusercontent.com/44624138/189450863-59c8c298-2759-4753-9566-6db2631a02c8.png)

Reply:
- Please be sure your design matches the original Figma design for a better User Interface(UI) 😉.
Your Design | Figma Design
![Screenshot from 2022-09-09 22-45-13](https://user-images.githubusercontent.com/44624138/189451142-c20b816b-5718-4179-84e1-9ef1a242f3b3.png) | ![Screenshot from 2022-09-09 22-48-12](https://user-images.githubusercontent.com/44624138/189451144-c854ca14-c0ea-4249-8a8e-dbf337c55def.png)